providers/implementations/kdfs/pbkdf2.c

```diff
@@ -35,6 +35,21 @@
 #define KDF_PBKDF2_MAX_KEY_LEN_DIGEST_RATIO 0xFFFFFFFF
 #define KDF_PBKDF2_MIN_ITERATIONS 1000
 #define KDF_PBKDF2_MIN_SALT_LEN   (128 / 8)
+/* The Implementation Guidance for FIPS 140-3 says in section D.N
+ * "Password-Based Key Derivation for Storage Applications" that "the vendor
+ * shall document in the module’s Security Policy the length of
+ * a password/passphrase used in key derivation and establish an upper bound
+ * for the probability of having this parameter guessed at random. This
+ * probability shall take into account not only the length of the
+ * password/passphrase, but also the difficulty of guessing it. The decision on
+ * the minimum length of a password used for key derivation is the vendor’s,
+ * but the vendor shall at a minimum informally justify the decision."
```

Comment:
Nowhere in this wall of text does it mandate any specific minimum size.  All it is mandating is that the vendor document the minimum size and provide guess probabilities and justification for those probabilities.  I.e. documenting this in the security policy should be sufficient to meet this IG:

> Minimum length of 1 character.
> 
> Character set | Guess Probability | Justification
> :-- | :-: | :--
> lower case letters | $` \frac{1}{26} `$ | There are 26 lower case letters of which one is chosen at random
> mixed case letters | $` \frac{1}{52} `$ | There are 52 mixed case letters of which one is chosen at random
> letters and digits | $` \frac{1}{62} `$ | There are 62 mixed case letters and digits of which one is chosen at random